R/across.R

```diff
@@ -634,9 +654,10 @@ expand_if_across <- function(quo) {
   call[[1]] <- quote(across)
   quos <- expand_across(quo_set_expr(quo, call))
 
-  # Select all rows if there are no inputs
+  # Select all rows if there are no inputs for if_all(),
+  # but select no rows if there are no inputs for if_any().
   if (!length(quos)) {
-    return(list(quo(TRUE)))
+    return(list(quo(empty)))
```

Comment:
```suggestion
    return(list(quo(!!empty)))
```

@krlmlr is right but i still think that having the `empty` variable is a little clearer, so here is the change you can make

Reply:
That makes sense. Thanks!